src/qt/askpassphrasedialog.cpp

```diff
@@ -104,10 +104,15 @@ void AskPassphraseDialog::accept()
             // Cannot encrypt with empty passphrase
             break;
         }
-        QMessageBox::StandardButton retval = QMessageBox::question(this, tr("Confirm wallet encryption"),
-                 tr("Warning: If you encrypt your wallet and lose your passphrase, you will <b>LOSE ALL OF YOUR BITCOINS</b>!") + "<br><br>" + tr("Are you sure you wish to encrypt your wallet?"),
-                 QMessageBox::Yes|QMessageBox::Cancel,
-                 QMessageBox::Cancel);
+        QMessageBox msgBoxConfirm(QMessageBox::Question,
+                                  tr("Confirm wallet encryption"),
+                                  tr("Warning: If you encrypt your wallet and lose your passphrase, you will <b>LOSE ALL OF YOUR BITCOINS</b>!<br><br>"
+                                     "Are you sure you wish to encrypt your wallet?"),
+                                  QMessageBox::Cancel | QMessageBox::Yes, this);
+        msgBoxConfirm.button(QMessageBox::Yes)->setText(tr("Continue"));
+        msgBoxConfirm.button(QMessageBox::Cancel)->setText(tr("Go back"));
```

Comment:
I'm not sure which label, "Go back" or just "Back", is more expected in user experience.

ping @GBKS

Reply:
I think both are fine. Personally, I would go for "Back", as the word "Go" doesn't add any additional information.

Reply:
Changed to "Back"